lib/python/rose/config_editor/panelwidget/summary_data.py

```diff
@@ -174,10 +174,8 @@ def _get_control_widget_hbox(self):
 
     def update_tree_model(self):
         """Construct a data model of other page data."""
-        sub_sect_names = self.sections.keys()
-        sub_var_names = []
         self.var_id_map = {}
-        for section, variables in self.variables.items():
+        for variables in self.variables.values():
```

Comment:
Whilst checking through the changes to this file I've spotted three other single-character names for statement variables between this and the next line change ('i' starting line 200, 'j' starting 203 and another 'i' starting 221) that should be changed to comply with pylint.